tpu-client-next/src/transaction_batch.rs

```diff
@@ -1,29 +1,38 @@
 //! This module holds [`TransactionBatch`] structure.
 
-use solana_time_utils::timestamp;
+use {solana_time_utils::timestamp, tokio_util::bytes::Bytes};
 
 /// Batch of generated transactions timestamp is used to discard batches which
 /// are too old to have valid blockhash.
 #[derive(Clone, PartialEq)]
 pub struct TransactionBatch {
     wired_transactions: Vec<WiredTransaction>,
+    // Time of creation of this batch, used for batch timeouts
     timestamp: u64,
 }
 
-type WiredTransaction = Vec<u8>;
+type WiredTransaction = Bytes;
 
 impl IntoIterator for TransactionBatch {
-    type Item = Vec<u8>;
+    type Item = Bytes;
     type IntoIter = std::vec::IntoIter<Self::Item>;
     fn into_iter(self) -> Self::IntoIter {
         self.wired_transactions.into_iter()
     }
 }
 
 impl TransactionBatch {
-    pub fn new(wired_transactions: Vec<WiredTransaction>) -> Self {
+    pub fn new<T>(wired_transactions: Vec<T>) -> Self
+    where
+        T: AsRef<[u8]> + Send + 'static,
+    {
+        let txs = wired_transactions
+            .into_iter()
+            .map(|v| Bytes::from_owner(v))
+            .collect();
+
         Self {
-            wired_transactions,
+            wired_transactions: txs,
```

Comment:
So in this case we can do
```suggestion
            wired_transactions,
```